packages/client-account/src/methods/accountCopyIndex.ts

```diff
@@ -58,7 +58,20 @@ export const accountCopyIndex = (
     )
     .then(() => Promise.resolve());
```

Comment:
or even 
```suggestion
    );
```

(because you're chaining it later)